src/applications/personalization/dashboard/containers/MessagingWidget.jsx

```diff
@@ -25,42 +24,42 @@ function recordDashboardClick(product) {
 class MessagingWidget extends React.Component {
   componentDidMount() {
     if (this.props.canAccessMessaging) {
-      this.props.fetchRecipients();
-      this.props.fetchFolder(0, { page: 1, sort: '-sent_date' });
+      this.props.fetchInbox();
     }
   }
 
   render() {
+    const { recipients, canAccessMessaging } = this.props;
+    let { messages } = this.props;
+    messages = messages || [];
+    let content;
+
+    if (!canAccessMessaging || (recipients && recipients.length === 0)) {
```

Comment:
I'm honestly not sure. I actually think we could probably just remover the checks on `recipients` since it looks like the changes to `componentDidMount` mean that `recipients` is never going to be set